agent/config/pbench-agent.cfg.example

```diff
@@ -40,8 +40,16 @@ interval = 30
 # pandas-package = python-pandas
 # Fedora has python2-pandas and python3-pandas
 # pandas-package = python2-pandas
+stockpile_dependencies = git, ansible
 
 [config]
 path = %(pbench_install_dir)s/config
 files = pbench-agent.cfg
 
+[stockpile]
+stockpile_user = root 
+local_remote_user = root 
+host_remote_user = root
+stockpile_path = /var/lib/pbench-agent
```

Comment:
It looks as if `stockpile_path` is optional in the code, with a hard-coded default of `/var/lib/pbench-agent`.  Can you confirm, and if so, make this a required parameter with a comment that describes why `/var/lib/pbench-agent` is used, and remove the hard-coded default in the code?

Reply:
You are right @portante , it's optional. Removed the hard-coded default, we use $pbench_run defined in the config if it's not set.